### DISCUSSION TITLE
Discussions Topics for Nov 8, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by clediggins-usds** [CDCgov/prime-reportstream#2944] [Show and tell]
Discussion Topics for Nov 8, 2021 Engineering Sync

**Comment by clediggins-usds** [comment]
Talk about how much cleanup to include in feature branches vs making separate tickets as they are found to be addressed as a different scope of work.

**Comment by carlosfelix2** [comment]
Intializing the DB with lookuptables before the schema check

**Comment by whytheplatypus** [comment]
Using [detekt](https://detekt.github.io/detekt/) or something like it to handle enforcing things like kdocs
